reckoner/chart.py

```diff
@@ -243,28 +234,39 @@ def install(self, default_namespace=None, default_namespace_management={}, conte
             self._context = context
         # Try to run the install process for mark the result as failed
 
-        try:
-            self.manage_namespace()
+        # TODO: Improve error handling of a repository installation
```

Comment:
Should we open an issue for this in order to track it?

Reply:
https://github.com/FairwindsOps/reckoner/issues/209